### DISCUSSION TITLE
Custom favicon + unread display choice

### DISCUSSION THREAD

**Discussion by niawag** [FreshRSS/FreshRSS#6065] [Ideas]
Hi and thanks a lot for FreshRSS, I'm coming from Feedly and using FreshRSS with RSS Proxy to create feeds for sites without it is great ! 
(a part of my question is discussed here: https://github.com/FreshRSS/FreshRSS/discussions/4734 but there was no solution at the time)

I love dark themes and especially the Nord one and I would like to know if the favicon of FreshRSS (not the favicons of feeds) could follow the chosen theme. So both following icons may be the same:
Favicon:
![image](https://github.com/FreshRSS/FreshRSS/assets/5684014/e918128b-0d68-4b68-8ead-89f30366141a)
Logo:
![image](https://github.com/FreshRSS/FreshRSS/assets/5684014/2d2448f1-9087-45e5-bcbd-df1000546ac4)
Is that currently possible ? If not, would it be with docker shenanigans (nothing too fancy, just a volume mouting custom_favicon.ico to favicon.ico) ?

Another related question, is it possible to hide the number of unread posts on the favicon, I get that many like that but I'd rather hide it. I've searched for a settings in the WebUI and the config file without success.

Keep up the good work !

**Comment by Alkarex** [comment]
Hello,
There is no option to change the favicon at the moment, but PR welcome.
Yes, it can be done manually, but there is more than one file:

https://github.com/FreshRSS/FreshRSS/blob/86631d24b9b41e35cd915d253a2040bdb8dbbfba/app/layout/layout.phtml#L23-L26

There is also some logic for the dynamic favicon, which might have to be checked if the favicon type or change change:

https://github.com/FreshRSS/FreshRSS/blob/86631d24b9b41e35cd915d253a2040bdb8dbbfba/p/scripts/main.js#L1855-L1894

**Reply by niawag** [reply]
Hi and thanks for your quick answer. I'd be glad to make a PR but I'm not a web expert. I'll see if I can manage it on my own. Here is how I think it should be done, can you check if I'm missing steps or if you'd be doing it differently:
- add two settings in config.php : 
     - favicon: "default" or "custom name"
     - dynamic favicon: "true" or "false"
- change "FreshRSS/app/layout/layout.phtml" to take car of this change
- change "FreshRSS/p/scripts/main.js" to make the function optionnal 

Another question, is config.php already "loaded" (or available) in both files ?

Edit:
I think I can copy what was done in this commit, there is changes to config.php, and examples on how to use new settings in other files: https://github.com/FreshRSS/FreshRSS/commit/d3966befafab3fe65e753d0ab2166a8bf5fb0d75

**Comment by SkilledAlpaca** [comment]
I found this through a quick search, was or has there been any progress on being able to disable the Unread counter for the favicon?

**Reply by Alkarex** [reply]
Nope
